### PR TITLE
Metadata file generated with the JOSE script and lightly edited

### PR DESCRIPTION
**Pull request by mbonsma**
This is the metadata file that was generated with the JOSE-provided [script](https://gist.github.com/arfon/478b2ed49e11f984d6fb). Addresses #37.

We will need to verify before submission:
* Project title
* Version number after https://github.com/UofTCoders/rcourse/issues/278 is taken care of

**Joel** - which email address do you want on here? **Lindsay** - do you have an ORCID?

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      "affiliation": "Department of Physics, University of Toronto"
+    }
+  ],
+  "identifier": "10.5281/zenodo.2335179",
+  "codeRepository": "https://github.com/UofTCoders/rcourse",
+  "datePublished": "2019-01-03",
+  "dateModified": "2019-01-03",
+  "dateCreated": "2019-01-03",
+  "description": "This material contains modular participatory live-coding lectures covering statistics and data analysis for ecology and reproducible quantitative methods in R. Statistical analysis, modelling, simulation, and data analysis are essential skills for applying ecology concepts to data. This material is designed to meet a growing demand for reproducible, openly accessible, analytically thorough, and well documented science.",
+  "keywords": "R, programming, coding, ecology, statistics, differential equations, model, regression, population dynamics",
```
Comment (aays) [inline codemeta.json]: Maybe s/model/modelling?

File: codemeta.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+  "@context": "https://raw.githubusercontent.com/codemeta/codemeta/master/codemeta.jsonld",
+  "@type": "Code",
+  "author": [
+    {
+      "@id": "0000-0003-4169-2616",
+      "@type": "Person",
+      "email": "lwjohnst@ph.au.tk",
```
Comment (lwjohnst86) [inline codemeta.json]: s/tk/dk

File: codemeta.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      "name": "Luke Johnston",
+      "affiliation": "Department of Nutritional Sciences, University of Toronto, Department of Public Health, Aarhus University"
```
Comment (lwjohnst86) [inline codemeta.json]: "Toronto; Department of Public Health"

File: codemeta.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  "identifier": "10.5281/zenodo.2335179",
+  "codeRepository": "https://github.com/UofTCoders/rcourse",
+  "datePublished": "2019-01-03",
+  "dateModified": "2019-01-03",
+  "dateCreated": "2019-01-03",
+  "description": "This material contains modular participatory live-coding lectures covering statistics and data analysis for ecology and reproducible quantitative methods in R. Statistical analysis, modelling, simulation, and data analysis are essential skills for applying ecology concepts to data. This material is designed to meet a growing demand for reproducible, openly accessible, analytically thorough, and well documented science.",
+  "keywords": "R, programming, coding, ecology, statistics, differential equations, model, regression, population dynamics",
+  "license": "MIT, CC-BY-4.0",
+  "title": "A graduate student-led participatory live-coding quantitative methods course in R: Experiences on initiating, developing, and teaching",
+  "version": "v2.0.0"
```
Comment (lwjohnst86) [inline codemeta.json:78]: This version will be updated after we've tidied up the course repo more

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**